packages/admin/cms-admin/src/preview/site/SitePreview.tsx

```diff
@@ -55,11 +56,11 @@ function SitePreview({ resolvePath, logo = <CometColor sx={{ fontSize: 32 }} />
     //use case for resolvePath: i18n urls, for example `/${scope.language}${path}`;
     //changes during navigation within site (iframe bridge reports new path)
     const [sitePath, setSitePath] = useSearchState("sitePath", (v) => {
-        if (v) {
-            return v;
-        } else {
-            return resolvePath ? resolvePath(initialPath, scope) : initialPath;
+        if (!v) {
+            v = resolvePath ? resolvePath(initialPath, scope) : initialPath;
         }
+        const url = new URL(v, siteConfig.url); // prevents phishing attacks
```

Comment:
Is it correct to use the site config url here? Wouldn't this be a different url? E.g. live site: `https://example.com`, site preview: `https://site-preview.example.com`